### PR TITLE
Expand description for Appwrite Function timeouts and variables.

### PR DESCRIPTION
**Pull request by gewenyu99**
Screen Shot 2022-07-15 at 3 28 59 PM

Add a detailed description of how to increase timeout and global maximum for execution timeout.

Related to: https://github.com/appwrite/appwrite/pull/3550

### CODE REVIEW

**Review by stnguyen90 — APPROVED** [review]
LGTM!

File: app/views/docs/functions.phtml [hunk 1]
```diff
@@ -648,7 +648,9 @@ class MainActivity : AppCompatActivity() {
 
 <p>When triggering a Cloud Function execution from the client, your users will be limited to a specific amount of execution per minute to make sure your Appwrite server is not being abused. The default limit is 60 calls per 1 minute. You can edit this limit using the server <a href="/docs/environment-variables#functions">environment variables</a>.</p>
 
-<p>The response size of a Cloud Function is limited to 1MB. Reponses larger than 1MB should be handled using Appwrite's Databases or Storage service.</p>
+<p>Function execution times out after 15 seconds by default. This timeout can be configured per function in a function's settings or in <span class="tag">appwrite.json</span> up to a global maximum of 900 seconds. The global timeout maximum can be further increased by changing the <span class="tag">_APP_FUNCTIONS_TIMEOUT</span> environment variable.</p>
+
+<p>The response size of a Cloud Function is limited to 1MB. Responses larger than 1MB should be handled using Appwrite's Databases or Storage service.</p>
```
Comment (eldadfux) [inline app/views/docs/functions.phtml:653]:
```suggestion
<p>The response size of a Function is limited to 1MB. Responses larger than 1MB should be handled using Appwrite's Databases or Storage service.</p>
```

**Review by eldadfux — CHANGES_REQUESTED** [review]
I think this PR is introducing future challenges and forces self-hosting on readers. Self-hosting notes should be separate. For an average Cloud user/developer any comment about environment variables will be EXTREMELY confusing.

**Comment by gewenyu99** [comment]
Screen Shot 2022-07-18 at 4 23 15 PM

@eldadfux I tried to address your comments by splitting the writing into different sections.

File: app/views/docs/functions.phtml [hunk 2]
```diff
@@ -646,9 +646,13 @@ class MainActivity : AppCompatActivity() {
 
 <p>Appwrite allows your project's end-users to execute Cloud Functions using client API or your client SDK. Execution is permitted to any user who has been granted the "execute" permission in your Cloud Functions settings page. The execution permission can accept any of the typical Appwrite <a href="/docs/permissions">permission types</a>.</p>
 
-<p>When triggering a Cloud Function execution from the client, your users will be limited to a specific amount of execution per minute to make sure your Appwrite server is not being abused. The default limit is 60 calls per 1 minute. You can edit this limit using the server <a href="/docs/environment-variables#functions">environment variables</a>.</p>
+<p>To prevent abuse, each user is rate limited to making 60 calls per minute to the Functions API. Each execution has a default timeout of 15 seconds to prevent hanging functions from blocking resources. This timeout can be configured per function in a function's settings or in <span class="tag">appwrite.json</span> for up to 900 seconds.</p>
```
Comment (christyjacob4) [inline app/views/docs/functions.phtml:649]: We must also mention that rate limits are not applicable to server SDKs authenticated with an API key allowing for usecases that need to by pass the default rate limit

Comment (gewenyu99) [reply app/views/docs/functions.phtml:649]: Yes! This is important, great catch! I didn't know this :P

**Comment by gewenyu99** [comment]
Screen Shot 2022-07-25 at 5 05 39 PM
@christyjacob4 Addressed.

**Review by christyjacob4 — APPROVED** [review]
Looks good 👌

**Pull request merged**